pkgs/applications/networking/instant-messengers/dino/default.nix

```diff
@@ -13,17 +13,18 @@
 , pcre
 , qrencode
 , icu
+, gspell, srtp, libnice, gnutls, gst_all_1
```

Comment:
This may not be right, but as I was working on upgrade myself,   and noticed that other derivations refer to gstreamer, et. al. directly, e.g. in `top-level-packages.nix`:

```
    monado = callPackage ../applications/graphics/monado {
    ▏ inherit (gst_all_1) gstreamer gst-plugins-base;
    };
```

Reply:
Following maybe required too for gstreamer plugins to be available in environment:

```
    preFixup = ''
      gappsWrapperArgs+=(--prefix GST_PLUGIN_SYSTEM_PATH : "$GST_PLUGIN_SYSTEM_PATH")
    '';
```

Reply:
I don't think gappsWrapperArgs is needed, since this derivation uses wrapGAppsHook, which sets GST_PLUGIN_SYSTEM_PATH: https://github.com/NixOS/nixpkgs/blob/9a3cfff2f9035ca0bd54ff13b608c4492eca6be3/pkgs/build-support/setup-hooks/wrap-gapps-hook/wrap-gapps-hook.sh#L34-L39

Reply:
> This may not be right, but as I was working on upgrade myself, and noticed that other derivations refer to gstreamer, et. al. directly, e.g. in `top-level-packages.nix`:
> 
> ```
>     monado = callPackage ../applications/graphics/monado {
>     ▏ inherit (gst_all_1) gstreamer gst-plugins-base;
>     };
> ```

I don't quite understand the implications of this, besides inputting only the required parts of gst_all_1 to the package, but following consensus seems pretty safe.